conferences/getcurrent.go

```diff
@@ -0,0 +1,86 @@
+package conferences
+
+import (
+	"context"
+	"fmt"
+
+	"encore.dev/storage/sqldb"
+)
+
+// GetCurrentByEventParams defines the inputs used by the GetCurrentByEvent API method
+type GetCurrentByEventParams struct {
+	EventID uint32
+}
+
+// GetCurrentByEventResponse defines the output returned by the GetCurrentByEvent API method
+type GetCurrentByEventResponse struct {
+	Event Event
+}
+
+// GetCurrentByEvent retrieves the current conference and event information for a specific event
+// encore:api public
+func GetCurrentByEvent(ctx context.Context, params *GetCurrentByEventParams) (*GetCurrentByEventResponse, error) {
+
+	rows, err := sqldb.Query(ctx,
+		`SELECT event.*,
```

Comment:
I would encourage you to add explicit list of columns, otherwise when someone adds columns your code might break due to lack of receivers.